documentation/how-to/create-gateway-service.md

```diff
@@ -1,43 +1,49 @@
 ---
-title: Create a Service
+title: Create a Gateway Service
 ---
 
 <!-- overview -->
 
-In Kong, the API gateway which underlies the API Services Portal, a {{ glossary_tooltip term_id="service" text="service" }}
-is an entity representing an external upstream API or microservice. 
-Service configuration allows Kong to manage, route, and handle requests to upstream services.
+In Kong, the API gateway which underlies the API Services Portal, a
+{{ glossary_tooltip term_id="gateway-service" }} is an entity representing an
+external upstream API or microservice. Service configuration allows Kong to
+manage, route, and handle requests to upstream services.
 
-Service configuration is stored in a GatewayService object in the API Services
-Portal. GatewayServices can be crafted from a template or generated from an
+Service configuration is stored in a Gateway Service object in the API Services
+Portal. Gateway Services can be crafted from a template or generated from an
 OpenAPI specification, both of which are explained below.
 
-Once you have created a GatewayService, its functionality can be extended with our
-[Supported Plugins](/reference/plugins/AVAILABLE-PLUGINS.md). They are essentially modular components 
-that can be added to Kong to add new features or alter the behavior of existing features.
+Once you have created a Gateway Service, its functionality can be extended with
+our [Supported Plugins](/reference/plugins/AVAILABLE-PLUGINS.md). They are
+essentially modular components that can be added to Kong to add new features or
+alter the behavior of existing features.
 
 <!-- ## Declarative Configuration -->
-<!-- this could be a good place to add in a general introduction to declarative syntax -->
+<!-- this could be a good place to add in a general introduction to declarative 
+syntax -->
 
 !!! warning "Upstream service setup"
     If your upstream services run on Platform
     Service's Silver or Gold OpenShift cluster, you will need to
     configure the network polices to allow access from the API Gateway.
-    See the [Upstream service setup](/how-to/upstream-services.md) guide for more information.
+    See the [Upstream service setup](/how-to/upstream-services.md) guide for
+    more information.
 
 ## Before you begin
 
 - [Install gwa CLI](/how-to/gwa-install.md)
-- [Create a Namespace](/reference/gwa-commands.md#namespacecreate)
+- [Create a Gateway](/reference/gwa-commands.md#gatewaycreate)
```

Comment:
All references to Create a Namespace/Gateway can be updated to point to /how-to/create-gateway.md
(need to pull changes from `ns-to-gw`)